CHANGELOG.md

```diff
@@ -2,17 +2,25 @@
 
 ## 0.0.11
 
-- BREAKING CHANGES: Keep a backup of your VIN & MAC Addr
+### Breaking change & Upgrade Instruction
+- No need to provide MAC addresses anymore for presence detection
+- **BREAKING CHANGE - save config before update**: Now supports **list** of VINS. You will need to adjust configuration. Existing **entities** from v0.0.10f will not be affected.
```

Comment:
Do we want to specify `of VINS and MAC addrs` ?

Reply:
yes done